.bbtravis.yml

```diff
@@ -25,7 +25,7 @@ env:
   - TWISTED=latest SQLALCHEMY=latest TESTS=js HYPER_SIZE=m1
   - TWISTED=latest SQLALCHEMY=latest TESTS=smokes HYPER_SIZE=m2
 
-  - TWISTED=16.1.0 SQLALCHEMY=latest TESTS=trial
+  - TWISTED=17.9.0 SQLALCHEMY=latest TESTS=trial
```

Comment:
I think TWISTED 17.9 is generating those StopIteration warnings

Reply:
No, the  StopIteration warnings were being caused by sqlalchemy 0.8.0.
I fixed those warnings here: https://github.com/buildbot/buildbot/pull/4499